scripts/deploy-prod-sample-module.js

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+const { argv } = require('yargs');
+
+const {
+  sanitizeEnvVars,
+  nginxOriginStaticsRootDir,
+  npmInstall,
+  npmProductionBuild,
+  getGitSha,
+} = require('./utils');
+
+const {
+  modulePath,
+  skipBuild,
+  skipInstall,
+  bundleStaticsOrigin = 'https://sample-cdn.frank',
+} = argv;
+
+const sanitizedEnvVars = sanitizeEnvVars();
+const nginxOriginStaticsModulesDir = path.resolve(nginxOriginStaticsRootDir, 'modules');
+const originModuleMapPath = path.resolve(nginxOriginStaticsRootDir, 'module-map.json');
+
+const getPreBuiltModuleInfo = (pathToModule) => {
+  const pkgPath = path.resolve(pathToModule, 'package.json');
+  // eslint-disable-next-line import/no-dynamic-require, global-require
+  const { name: moduleName, version: moduleVersion } = require(pkgPath);
+
+  const pathToBundleIntegrityManifest = path.resolve(`${pathToModule}/bundle.integrity.manifest.json`);
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const integrityDigests = require(pathToBundleIntegrityManifest);
+
+  return { moduleName, moduleVersion, integrityDigests };
+};
+
+const buildModule = async (pathToModule) => {
+  const pkgPath = path.resolve(pathToModule, 'package.json');
+  // eslint-disable-next-line import/no-dynamic-require, global-require
+  const { name: moduleName, version: moduleVersion } = require(pkgPath);
+
+  if (!skipInstall) {
+    await npmInstall({
+      directory: pathToModule,
+      moduleName,
+      moduleVersion,
+      envVars: sanitizedEnvVars,
+    });
+  }
+
+  await npmProductionBuild({
+    directory: pathToModule,
+    moduleName,
+    moduleVersion,
+    envVars: sanitizedEnvVars,
+  });
```

Comment:
Is this needed to override the development build created after install?

Reply:
yeah this ensures the build is run with `NODE_ENV=production`
https://github.com/americanexpress/one-app/pull/308/files#diff-db5dfce483b0c43f844c909e42e27356R72